tools/go-cli/go-whisk-cli/main.go

```diff
@@ -20,17 +20,30 @@ import (
     "fmt"
     "os"
     "reflect"
+    "time"
 
     "../go-whisk/whisk"
     "../go-whisk-cli/commands"
 )
 
+// CLI_BUILD_TIME holds the time of the CLI build.  During production builds,
+// this value will be overwritten via the command:
+//     go build -ldflags "-X main.CLI_BUILD_TIME=nnnnn"   // nnnnn is the new timestamp
+var CLI_BUILD_TIME string = "not set"
+
 var cliDebug = os.Getenv("WSK_CLI_DEBUG")  // Useful for tracing init() code
 
 func init() {
     if len(cliDebug) > 0 {
         whisk.SetDebug(true)
     }
+
+    // Initialize a default CLI build timestamp value only if the value was not
+    // set via the go linker flags.
+    if CLI_BUILD_TIME == "not set" {
+        CLI_BUILD_TIME = time.Now().Format("2006-01-02T15:04:05-07:00")
```

Comment:
I'm not sure we should put a fake timestamp there. If for some reason a defect arose in the Docker build process preventing the timestamp from being written into the CLI_BUILD_TIME variable, we would never know. Without writing a default timestamp the automation would catch such a defect.